### PR TITLE
Scoped components

### PR DESCRIPTION
**Pull request by martind-bjss**
Removing shadow DOM for better a11y support.

<details>
  <summary>📦 Published PR as canary version: <code>0.0.9--canary.39.389456a.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @ukho/admiralty-angular@0.0.9--canary.39.389456a.0
  npm install @ukho/admiralty-core@0.0.9--canary.39.389456a.0
  # or 
  yarn add @ukho/admiralty-angular@0.0.9--canary.39.389456a.0
  yarn add @ukho/admiralty-core@0.0.9--canary.39.389456a.0
  ```
</details>

### CODE REVIEW

File: packages/core/src/components/colour-block/colour-block.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ButtonVariant } from '../button/button.types';
 @Component({
   tag: 'admiralty-colour-block',
   styleUrl: 'colour-block.scss',
-  shadow: false,
+  scoped: false,
```
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/colour-block/colour-block.tsx:7]: No - fixed now

File: packages/core/src/components/checkbox/checkbox.spec.tsx
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/checkbox/checkbox.spec.tsx:1]: This was the file extension added by `stencil generate`.  However I have changed this to `.ts`

File: packages/core/src/components/header-sub-menu-item/header-sub-menu-item.spec.tsx
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/header-sub-menu-item/header-sub-menu-item.spec.tsx:1]: This was the file extension added by `stencil generate`.  However I have changed this to `.ts`

File: packages/core/src/components/textarea/textarea.spec.tsx
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/textarea/textarea.spec.tsx:1]: This was the file extension added by `stencil generate`.  However I have changed this to `.ts`

File: packages/core/src/components/select/select.spec.tsx
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/select/select.spec.tsx:1]: This was the file extension added by `stencil generate`.  However I have changed this to `.ts`

File: packages/core/src/components/type-ahead/test/type-ahead.spec.tsx
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/type-ahead/test/type-ahead.spec.tsx:1]: This was the file extension added by `stencil generate`.  However I have changed this to `.ts`

File: packages/core/src/components/label/label.tsx [hunk 2]
```diff
@@ -20,6 +22,13 @@ export class LabelComponent {
    */
   @Prop() for: string;
 
+  // componentWillRender() {
+  //   console.log('el', this.el);
+  //   if (this.for) {
+  //     this.el.setAttribute('for', this.for);
+  //   }
+  // }
+
```
[the remark this answers is not in the document]
Comment (martind-bjss) [reply packages/core/src/components/label/label.tsx]: deleted

**Review by UKHOGreg — APPROVED** [review]

**Pull request merged**